.gitignore

```diff
@@ -0,0 +1,58 @@
+.eslintcache
+.cache
+scalastyle-output.xml
+.classpath
+.idea/*
+!.idea/vcs.xml
+.metadata
+.settings
+.project
+.version.properties
+filter.properties
+logs.zip
+target
+tmp
+*.class
+*.iml
+*.swp
+*.jar
+*.zip
+*.log
+*.pyc
+.DS_Store
+build-target
+flink-runtime-web/web-dashboard/node/
+flink-runtime-web/web-dashboard/node_modules/
+flink-runtime-web/web-dashboard/web/
+flink-python/dist/
+flink-python/apache-flink-libraries/dist/
+flink-python/build/
+flink-python/apache-flink-libraries/build
+flink-python/pyflink.egg-info/
+flink-python/apache_flink.egg-info/
+flink-python/apache-flink-libraries/apache_flink_libraries.egg-info/
+flink-python/docs/_build
+flink-python/.tox/
+flink-python/dev/download
+flink-python/dev/.conda/
+flink-python/dev/log/
+flink-python/dev/.stage.txt
+flink-python/.eggs/
+flink-python/apache-flink-*.dev0/
+flink-python/apache-flink-libraries/apache_flink_libraries-*.dev0/
+flink-python/**/*.c
+flink-python/**/*.so
+atlassian-ide-plugin.xml
```

Comment:
Why did we add all of this?

Reply:
Just copy pasted from flink repo

Reply:
Maybe this is a bit too much. I would suggest to extend this file as we go and see need for it.